test_ext_imgconverter.py fails if ImageMagick is not installed

### Describe the bug

Hi! First ever contributor here, and currently learning Python :)

When cloning the repository and executing the test suite for the first time, `test_ext_imgconverter.py` failed because the `svgimg.svg` was not converted to `.png` as expected. The root cause was not clear.

After digging, I understood the reason was I didn't have ImageMagick installed (I'm on macOS), therefore `ImagemagickConverter.is_available()` returned `False` and the image conversion was skipped.

**I think I can at least add a comment in the test, and a precision in the doc to avoid confusing other new contributors.
However, maybe the test should be skipped if ImageMagick is not installed. What do you think?**

### How to Reproduce

Prerequisite: Not having ImageMagick installed.

Then from a python 3.10 venv:
```
pytest tests/test_ext_imgconverter.py
```


### Expected behavior

Test should pass.

### Your project

sphinx

### Screenshots

Here is the test failure output:
```
AssertionError: assert '\\sphinxincludegraphics{{svgimg}.png}' in '%% Generated by Sphinx.\n\\def\\sphinxdocclass{report}\n\\documentclass[letterpaper,10pt,english]{sphinxmanual}\n\\if...n\\noindent\\sphinxincludegraphics{{img}.pdf}\n\n\n\n\\renewcommand{\\indexname}{Index}\n\\printindex\n\\end{document}'

tests/test_ext_imgconverter.py:19: AssertionError
-------------------------------------------------------------------------- Captured stdout teardown ----------------------
# testroot: root
# builder: latex
# srcdir: /private/var/folders/3d/p4l2y35x3417mqqwvs3txmw0dprkcb/T/pytest-of-maximed/pytest-49/ext-imgconverter
# outdir: /private/var/folders/3d/p4l2y35x3417mqqwvs3txmw0dprkcb/T/pytest-of-maximed/pytest-49/ext-imgconverter/_build/latex
# status:
Running Sphinx v4.5.1+/e7cba3516
building [latex]: all source files
updating environment: [new config] 1 added, 0 changed, 0 removed
reading sources... [100%] index
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
processing python.tex... index
resolving references...
done
writing... done
copying images... [100%] img.pdf
copying TeX support files... copying TeX support files...
done

# warning:
WARNING: convert command 'convert' cannot be run, check the image_converter setting: [Errno 2] No such file or directory: 'convert'
```

The warning at the end is actually the real reason of failure, but I didn't realize it right away. 😅 

### OS

macOS 11.6.2

### Python version

3.10.0

### Sphinx version

4.5.0

### Sphinx extensions

_No response_

### Extra tools

_No response_

### Additional context

_No response_

--- a/tests/test_ext_imgconverter.py
+++ b/tests/test_ext_imgconverter.py
@@ -1,12 +1,26 @@
 """Test sphinx.ext.imgconverter extension."""
 
-import os
+import subprocess
+from subprocess import PIPE
 
 import pytest
 
 
+@pytest.fixture
+def if_converter_found(app):
+    image_converter = getattr(app.config, 'image_converter', '')
+    try:
+        if image_converter:
+            subprocess.run([image_converter, '-version'], stdout=PIPE, stderr=PIPE)  # show version
+            return
+    except OSError:  # No such file or directory
+        pass
+
+    pytest.skip('image_converter "%s" is not available' % image_converter)
+
+
+@pytest.mark.usefixtures('if_converter_found')
 @pytest.mark.sphinx('latex', testroot='ext-imgconverter')
-@pytest.mark.xfail(os.name != 'posix', reason="Not working on windows")
 def test_ext_imgconverter(app, status, warning):
     app.builder.build_all()
 

